Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/AnalysisServerStatusAction.java

```diff
@@ -28,12 +28,16 @@ public AnalysisServerStatusAction() {
   public void actionPerformed(AnActionEvent e) {
     DartFeedbackBuilder builder = DartFeedbackBuilder.getFeedbackBuilder();
 
-    if (MessageDialogBuilder.yesNo(builder.title(), builder.prompt())
-          .icon(Messages.getQuestionIcon())
-          .yesText(builder.label())
-          .show() == Messages.YES) {
-      builder.sendFeedback(e.getProject());
+    if (builder.isPromptRequired()) {
```

Comment:
Forgot to mention that I agree with your concern here. I was thinking of adding a "Don't show this again" checkbox. But that means adding a preference and requires some code I hadn't gotten around to writing yet.